.github/workflows/ci.yml

```diff
@@ -45,23 +45,28 @@ jobs:
   test:
     name: Run ci-tests
     runs-on: ubuntu-latest
-    container:
-      image: golang:latest
     steps:
       - name: Checkout Code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.2' #
       - name: Make repo safe
         run: git config --global --add safe.directory /__w/SOARCA/SOARCA
       - name: Install swaggo
-        run:  go install github.com/swaggo/swag/cmd/swag@latest   
+        run:  go install github.com/swaggo/swag/cmd/swag@latest  
+        timeout-minutes: 12
+      - name: Start docker containers for test
+        run: docker-compose -f "docker/testing/httpbin-test/docker-compose.yml" up -d --build
       - name: Run tests
         run: |
-          apt update
-          apt install openssh-server -y
-          useradd sshtest
-          echo "sshtest:pdKY77qNxpI5MAizirtjCVOcm0KFKIs" | chpasswd
-          service ssh start
+          sudo apt update
```

Comment:
Why does it need sudo ?

Reply:
because it is not root anymore

Reply:
I removed the container section that required root